h/static/scripts/root-thread.js

```diff
@@ -27,6 +27,17 @@ var sortFns = {
 };
 
 /**
+ * Return true if a given `draft` is empty and can be discarded with losing
+ * any user input
+ */
+function isEmptyDraft(draft) {
```

Comment:
If there's an easy way to make `isEmpty` the responsibility of the drafts service, I think that would be preferable.